src/runtime/startup/c/wrapper.c

```diff
@@ -760,13 +788,8 @@ __attribute__((constructor)) static void init(void) {
         stage_pid = fork();
     } else {
         if ( config.nsFlags & CLONE_NEWPID ) {
-#ifdef NS_CLONE_NEWPID
```

Comment:
Why do we remove check for if PIDNS supported?

Reply:
I think CLONE_NEWPID is supported on system having NO_NEW_PRIVS :)